OpenRA.Game/Server/Server.cs

```diff
@@ -306,7 +306,7 @@ void ValidateClient(Connection newConn, string data)
 					IpAddress = ((IPEndPoint)newConn.Socket.RemoteEndPoint).Address.ToString(),
 					Index = newConn.PlayerIndex,
 					Slot = LobbyInfo.FirstEmptySlot(),
-					PreferredColor = handshake.Client.Color,
+					PreferredColor = handshake.Client.PreferredColor,
```

Comment:
I remember having a specific reason for using `Color` here, but I can't remember what it was.  Probably working around some even older bug that is no longer valid.